src/mocks/handlers.ts

```diff
@@ -15,4 +17,25 @@ export const handlers = [
 
     return res(ctx.status(200), ctx.json(supplement));
   }),
+
+  rest.post("/cart_items", async (req, res, ctx) => {
```

Comment:
What do you think about just using `/cart` for all of the REST Verbs instead of `/cart_items`?